README.md

```diff
@@ -1,70 +1,103 @@
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Other Important Information
+## Built With
 
-### `npm test`
+- Basic Language - JavaScript 
+- Framework - React
+- Other technologies/tools: 
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+``` create-react-app
+    > create-react-app
+    > webpack for bundling files
+    > Babel for code transpiling
+    > Git for version control
+    > Eslint for JavaScript linting
+    > Stylelint for style linting
+    > Jest for testing
+ ```
 
-### `npm run build`
+## Live Demo
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+- [Live Demo Link](https://kingsleyibe.github.io/bookstore/)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Getting Started
 
-### `npm run eject`
+To get a local copy up and running follow these simple example steps.
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+### Prerequisites
+- You should have basic knowledge of HTML.
+- You should have basic knowledge of JavaScript.
+- You should have basic knowledge of CSS.
+- You should have basic knowledge of webpack.
+- Also, a basic knowledge of git and github.
+- A code editor: I used VScode for this project but you can use any code editor of your choice.
+### Setup
+- follow steps on this link to set up VScode code editor: [setup vscode](https://www.freecodecamp.org/news/how-to-set-up-vs-code-for-web-development/)
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Install
+- [download](https://code.visualstudio.com/download) VScode.
+[Watch this video](https://www.youtube.com/watch?v=MlIzFUI1QGA) on how to install VScode on windows.
+- [Git]()
+- [Node]()
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+### Usage
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Clone project, run in your terminal:
 
-## Learn More
+```$ git clone https://github.com/KingsleyIbe/react-todos-app.git ```
```

Comment:
- [ ] Let's try to be thorough in our documentation as it may be of great help to users. This is not the correct link for cloning the repo. Kindly fix that. Here is the correct one: **`https://github.com/KingsleyIbe/bookstore.git`**

Reply:
Hey @TedLivist 

Thank you for your feedback.